src/main/java/uk/gov/hmcts/reform/jobscheduler/controllers/RootController.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.jobscheduler.controllers;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import static org.springframework.http.ResponseEntity.ok;
+
+/**
+ * Default endpoints per application.
+ */
+@RestController
+public class RootController {
+
+    /**
+     * Root GET endpoint.
+     *
+     * <p>Azure application service has a hidden feature of making requests to root endpoint when
+     * "Always On" is turned on.
+     * This is the endpoint to deal with that and therefore silence the unnecessary 404s as a response code.
+     *
+     * @return Welcome message from the service.
+     */
+    @GetMapping
+    public ResponseEntity<String> welcome() {
+        return ok("Welcome to spring-boot-template");
```

Comment:
`Welcome to Job scheduler` ?

Reply:
Btw CMC has done it this was https://github.com/hmcts/cmc-claim-store/pull/263/files#diff-7a539344c51609e77ea7e6822993678aR10

Reply:
Should firstly update [template](https://github.com/hmcts/spring-boot-template/pull/31/files). I'd say empty body for get method does not make sense. Should then respond with 204 as it's no content, but then again - GET method.

p.s. oops, missed slug copy